website/tutorial/running-an-ai-application.md

```diff
@@ -0,0 +1,136 @@
+2. In the Catalog section, click the **Download** icon corresponding to the model you want to use.
+   ![extension enabled](img/download-a-model.png)
+
+3. View the downloaded model on the same page.
+   ![extension enabled](img/model-downloaded.png)
+
+## Creating a service
+
+1. Click **AI Lab** in the left navigation pane.
+2. In the Services section, click **New Model Service**.
+   ![create a new model service](img/create-new-model-service.png)
+
+3. Select the model from the dropdown list.
+4. Optional: Edit the port number of the container that is created for the service.
+5. Click **Create service**.
+   ![create a new model service](img/create-a-service.png)
+
+6. Click **Open service details**.
+   ![create a new model service](img/open-service-details.png)
+
+7. View the service details, such as model name, server address, and client code.
+   ![service details page](img/service-details-page.png)
+
+8. Click the **Close** icon on the right side of the page.
+9. View the newly created service on the same page.
+   ![new service on the page](img/newly-created-service.png)
+
+10. Optional: View the service container by opening the **Containers** page from the left navigation pane.
+    ![new service container](img/running-service-in-a-container.png)
+
+## Optional: Creating a playground
+
+Perform this task only if you want to experiment with the model before using it to start your AI application.
+
+1. Click **AI Lab** in the left navigation pane.
+2. In the Playgrounds section, click **New Playground**.
+   ![create a new playground](img/new-playground.png)
+
+3. Optional: Enter the name of the playground.
+4. Select a model from the dropdown list.
+5. Click **Create playground**.
+   ![create playground button](img/create-a-playground.png)
+
+6. View the newly created playground on the same page.
+   ![newly created playground](img/newly-created-playground.png)
+
+7. Click the name of the playground to edit the model settings:
+
+   1. Edit the value of the model parameters.
+      ![editing model parameters](img/editing-model-parameters.png)
+
+   2. Click the **Edit system prompt** icon to add a system prompt and save it.
+      ![edit system prompt icon](img/edit-system-prompt-icon.png)
+
+   3. Enter your question in the prompt box and validate the answer.
+      ![defining a system prompt](img/defining-a-system-prompt.png)
+
+## Running a recipe: ChatBot AI application
+
+1. Click **AI Lab** in the left navigation pane.
+2. In the Recipes Catalog section, click **More details** in the ChatBot recipe tile. The recipe summary page opens.
+   ![more details of a recipe](img/more-details-of-recipe.png)
+
+3. Click **Start**. The Start recipe page opens.
+   ![starting a recipe](img/starting-a-recipe.png)
+
+4. Select the model you want to use with the recipe.
+5. Click **Start ChatBot recipe**.
+   ![starting a chatbot recipe](img/starting-a-chatbot-recipe.png)
+
+6. Click **Open details**.
```

Comment:
You could indicate that the previous operation can take a while (and perhaps briefly describe what happens behind the hood)